addons/my-ocular/discuss.js

```diff
@@ -1,33 +1,107 @@
 export default async function ({ addon, global, console, msg }) {
-  var lefts = document.querySelectorAll("div.postleft > dl");
-  lefts.forEach(async (i) => {
-    var username = i.children[0].innerText;
+  let posts = document.querySelectorAll(".blockpost");
 
-    var response = await fetch(`https://my-ocular.jeffalo.net/api/user/${username}`);
-    var data = await response.json();
-    var userStatus = data.status;
-    var color = data.color;
+  let ocularReactionColor = getComputedStyle(document.querySelector(".blockpost").querySelector(".box"))
+    .backgroundColor; // match post header
+  document.documentElement.style.setProperty("--ocular-reaction-color", ocularReactionColor); // match post header
+
+  let ocularReactionBGColor = getComputedStyle(document.querySelector(".blockpost").querySelector(".postleft"))
+    .backgroundColor; // match post header
+  document.documentElement.style.setProperty("--ocular-reaction-bg-color", ocularReactionBGColor); // match post header
```

Comment:
Please don't use `getComputedStyle` to do this: it doesn't update when scratchr2 or dark mode is enabled/disabled. Instead, include the default styles in the userstyle and override them in `scratchr2/forums.css` and `dark-www/experimental_forums.css`.

Reply:
aw. okay then.

Reply:
fixed